apps/docs/content/components/Section/index.mdx

```diff
@@ -0,0 +1,163 @@
+      'https://github.com/user-attachments/assets/a7340dea-2076-42da-87a9-057f953beb83',
+      'https://github.com/user-attachments/assets/a28110fd-d019-41a4-8f80-b49ae8895708',
+    ]}
+    backgroundImagePosition={['bottom -1px center', 'center']}
+    backgroundImageSize={['auto', 'cover']}
+  >
+    <SectionIntro align="center">
+      <SectionIntro.Heading size="3" size="3">
+        Multiple background images
+      </SectionIntro.Heading>
+      <SectionIntro.Description>
+        This section uses multiple layered background images.
+      </SectionIntro.Description>
+    </SectionIntro>
+  </Section>
+  <Section backgroundColor="default">
+    <SectionIntro align="center">
+      <SectionIntro.Heading size="4">Another section</SectionIntro.Heading>
+    </SectionIntro>
+  </Section>
+</ThemeProvider>
+```
+
+### With rounded corners
+
+```jsx live
+  <ThemeProvider colorMode="dark">
+    <Section backgroundColor="default"></Section>
+  </ThemeProvider>
+  <Section backgroundColor="default" rounded>
+    <Grid>
+      <Grid.Column>
+        <SectionIntro align="center">
+          <SectionIntro.Heading size="3">Rounded corners</SectionIntro.Heading>
+          <SectionIntro.Description>
+            This section has rounded top corners that overlap the previous
+            section.
+          </SectionIntro.Description>
+        </SectionIntro>
+      </Grid.Column>
+    </Grid>
+  </Section>
+```
+
+## Component props
+
+### Section
+
+| name                      | type                                                                                  | default    | required | description                                                                                                                                                   |
+| ------------------------- | ------------------------------------------------------------------------------------- | ---------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `paddingBlockStart`       | <PropTableValues values={[...SectionPaddingVariants]} commaSeparated  />              | `'normal'` | `false`  | Control the padding at the top of the section.                                                                                                                |
+| `paddingBlockEnd`         | <PropTableValues values={[...SectionPaddingVariants]} commaSeparated  />              | `'normal'` | `false`  | Control the padding at the bottom of the section.                                                                                                             |
+| `backgroundColor`         | <PropTableValues values={[...SectionBackgroundColors]} commaSeparated  /> \| `string` |            | `false`  | Add a background color. Use a predefined color, a CSS custom property, or a custom color value.                                                               |
```

Comment:
Looks like you might be missing the ResponsiveMap type from this table. Can we add that in please?